Fix javadoc under Java 8

```
Constructing Javadoc information...
Standard Doclet version 1.8.0_77
Building tree for all the packages and classes...
Generating /root/opensource/Decision/api/target/apidocs/com/stratio/decision/api/partitioner/HashPartitionerStrategy.html...
Generating /root/opensource/Decision/api/target/apidocs/com/stratio/decision/api/partitioner/IPartitionerStrategy.html...
Generating /root/opensource/Decision/api/target/apidocs/com/stratio/decision/api/partitioner/JenkinsHash.html...
Generating /root/opensource/Decision/api/target/apidocs/com/stratio/decision/api/partitioner/PartitionerStrategyFactory.html...
Generating /root/opensource/Decision/api/target/apidocs/com/stratio/decision/api/partitioner/PartitionerStrategyFactory.Strategy.html...
Generating /root/opensource/Decision/api/target/apidocs/com/stratio/decision/api/partitioner/package-frame.html...
Generating /root/opensource/Decision/api/target/apidocs/com/stratio/decision/api/partitioner/package-summary.html...
Generating /root/opensource/Decision/api/target/apidocs/com/stratio/decision/api/partitioner/package-tree.html...
Generating /root/opensource/Decision/api/target/apidocs/constant-values.html...
Generating /root/opensource/Decision/api/target/apidocs/com/stratio/decision/api/partitioner/class-use/JenkinsHash.html...
Generating /root/opensource/Decision/api/target/apidocs/com/stratio/decision/api/partitioner/class-use/PartitionerStrategyFactory.html...
Generating /root/opensource/Decision/api/target/apidocs/com/stratio/decision/api/partitioner/class-use/PartitionerStrategyFactory.Strategy.html...
Generating /root/opensource/Decision/api/target/apidocs/com/stratio/decision/api/partitioner/class-use/HashPartitionerStrategy.html...
Generating /root/opensource/Decision/api/target/apidocs/com/stratio/decision/api/partitioner/class-use/IPartitionerStrategy.html...
Generating /root/opensource/Decision/api/target/apidocs/com/stratio/decision/api/partitioner/package-use.html...
Building index for all the packages and classes...
Generating /root/opensource/Decision/api/target/apidocs/overview-tree.html...
Generating /root/opensource/Decision/api/target/apidocs/index-all.html...
Generating /root/opensource/Decision/api/target/apidocs/deprecated-list.html...
Building index for all classes...
Generating /root/opensource/Decision/api/target/apidocs/allclasses-frame.html...
Generating /root/opensource/Decision/api/target/apidocs/allclasses-noframe.html...
Generating /root/opensource/Decision/api/target/apidocs/index.html...
Generating /root/opensource/Decision/api/target/apidocs/help-doc.html...
2 errors
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary:
[INFO] 
[INFO] Stratio Decision ................................... SUCCESS [  0.845 s]
[INFO] Decision commons ................................... SUCCESS [  2.575 s]
[INFO] Stratio Decision examples .......................... SUCCESS [  2.105 s]
[INFO] Decision Siddhi extensions ......................... SUCCESS [ 19.708 s]
[INFO] Decision engine .................................... SUCCESS [ 16.072 s]
[INFO] Stratio Decision API ............................... FAILURE [ 23.774 s]
[INFO] Decision shell ..................................... SKIPPED
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 01:05 min
[INFO] Finished at: 2016-06-04T20:34:26+08:00
[INFO] Final Memory: 89M/779M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-javadoc-plugin:2.9.1:jar (attach-javadocs) on project api: MavenReportException: Error while creating archive:
[ERROR] Exit code: 1 - /root/opensource/Decision/api/src/main/scala/com/stratio/decision/api/partitioner/JenkinsHash.java:76: error: bad HTML entity
[ERROR] * <code>h = (h & hashmask(10));</code>
[ERROR] ^
[ERROR] /root/opensource/Decision/api/src/main/scala/com/stratio/decision/api/partitioner/JenkinsHash.java:80: error: malformed HTML
[ERROR] * for (int i = 0, h = 0; i < n; ++i) h = hash( k[i], h);
[ERROR] ^
[ERROR] 
[ERROR] Command line was: /opt/software/jdk1.8.0_77/jre/../bin/javadoc @options @packages
[ERROR] 
[ERROR] Refer to the generated Javadoc files in '/root/opensource/Decision/api/target/apidocs' dir.
[ERROR] -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoExecutionException
[ERROR] 
[ERROR] After correcting the problems, you can resume the build with the command
[ERROR]   mvn <goals> -rf :api
```

--- a/api/src/main/scala/com/stratio/decision/api/partitioner/JenkinsHash.java
+++ b/api/src/main/scala/com/stratio/decision/api/partitioner/JenkinsHash.java
@@ -73,11 +73,11 @@ public final class JenkinsHash {
      * <p>The best hash table sizes are powers of 2.  There is no need to do mod
      * a prime (mod is sooo slow!).  If you need less than 32 bits, use a bitmask.
      * For example, if you need only 10 bits, do
-     * <code>h = (h & hashmask(10));</code>
+     * <code>h = (h &amp; hashmask(10));</code>
      * In which case, the hash table should have hashsize(10) elements.
      *
      * <p>If you are hashing n strings byte[][] k, do it like this:
-     * for (int i = 0, h = 0; i < n; ++i) h = hash( k[i], h);
+     * for (int i = 0, h = 0; i &lt; n; ++i) h = hash( k[i], h);
      *
      * <p>By Bob Jenkins, 2006.  bob_jenkins@burtleburtle.net.  You may use this
      * code any way you wish, private, educational, or commercial.  It's free.

